app/Lio/ServiceProviders/HashidsServiceProvider.php

```diff
@@ -5,15 +5,11 @@
 
 class HashidsServiceProvider extends ServiceProvider
 {
-    public function boot()
+    public function register() 
     {
-        $this->app->bind('Hashids\Hashids', function() {
+        $this->app->bind(['Hashids\Hashids' => 'hashids'], function() {
```

Comment:
Also, `$app` should be passed into this function as the first param, and used instead of `$this->app`.